lib/mcb.rb

```diff
@@ -323,7 +323,11 @@ def env_to_azure_map
 
     def load_env_azure_settings(opts)
       if opts.key?(:env)
-        env_settings = env_to_azure_map.fetch(opts[:env].to_sym)
+        begin
+          env_settings = env_to_azure_map.fetch(opts[:env].to_sym)
+        rescue KeyError
+          raise KeyError, "The environment '#{opts[:env]}' could not be found, have you made sure to add it to your config/settings/#{ENV['RAILS_ENV']}.local.yml?"
+        end
```

Comment:
I'm curious why the `if opts.key?(:env)` doesn't make the `rescue` unreachable as I would have expected, do you know?

Reply:
(I was originally going to suggest changing the rescue to an if block, then I saw one and was confused)

Reply:
The `KeyError` arises as a result of `env_to_azure_map.fetch`, the `opts.key?` will only ensure that the options have specified an environment to switch to.

Reply:
Ah in which case I'd misunderstood the code as written so I definitely would switch the rescue to an if